index.html.haml

```diff
@@ -80,10 +80,10 @@ layout: base
           Drools Fusion (complex event processing features)
       %li
         %img(src = "headerFooter/flow_icon_32x.png")
-          %a(href="http://www.jboss.org/jbpm") jBPM (process/workflow integration for rule orchestration/flow)
+          %a(href="https://www.jboss.org/jbpm") jBPM (process/workflow integration for rule orchestration/flow)
```

Comment:
jboss.org/jbpm vs jbpm.org
It's not strictly related to the changes you did, but we should fix it anyway.